controllers/post/PostController.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace humhub\modules\smartVillage\controllers\post;
+
+use humhub\modules\content\components\ActiveQueryContent;
+use humhub\modules\content\components\ContentActiveRecord;
+use humhub\modules\content\models\ContentContainer;
+use humhub\modules\post\models\Post;
+use humhub\modules\rest\definitions\ContentDefinitions;
+use humhub\modules\rest\definitions\PostDefinitions;
+use humhub\modules\smartVillage\components\AuthBaseController;
+
+class PostController extends AuthBaseController
+{
+    public function actionFind(){
+        $query = Post::find()->joinWith('content')->orderBy(['content.created_at' => SORT_DESC])->readable();
```

Comment:
the `visibility` for all public also needs to be considered here for posts please

Reply:
Hi @donni106 
Okay, we will do it.

Reply:
so `Post::find()` already returns only publicly visible. nice. i have tested it with a private post, which was only returned for a logged in user that is member of the space where the post was posted in.